docs/documentation/updating-the-kit.md

```diff
@@ -1,129 +1,44 @@
-# Updating the kit
+# Updating to the latest version
 
-**Important note**
+1. Make a backup copy of your prototype folder.
 
-If you have made any changes outside the `app` folder, this process will destroy those changes. We will try and improve the update process to avoid this, but in the meantime you will need to make a note of your changes outside `app`, and add them back after updating.
+2. In your prototype, delete everything except the `app` and `.git` folders.
 
-## Steps
+   You may not be able to see the `.git` folder because files and folders that start with `.` are hidden by default.
 
-[Download the latest Prototype Kit](/docs/download) and unzip it.
+3. [Download the zip file of the latest version of the Prototype Kit](/docs/download).
+  
+4. Unzip the zip file.
+ 
+5. Copy all the folders except the `app` folder to your prototype.
 
-In your project, delete everything apart from the `app` and `.git` folder.
+6. Copy the `app/config.js` file to the `app` folder of your prototype.
 
-Copy everything from the latest kit to your project, apart from the `app` folder.
+7. Compare your new `config.js` file to the `config.js` file in the backup you made in step 1, and copy over anything you need to - for example your service name.
 
-Copy the config.js file from the `app` folder in the latest kit to the `app` folder of your prototype. If you've made any changes to the config.js file in your prototype then you'll need to re-enter them in the new version of the file e.g. the service name.
+8. Copy the `app/assets/sass/patterns` folder to your prototype.
 
-Check `\app\assets\sass\patterns` in the latest kit for any new patterns. Copy the files over to your prototype.
+9. Copy the `app/assets/sass/components` folder to your prototype.
 
-Check `\app\assets\sass\application.scss` in the latest kit to see if any changes have been made in the top section, above where it says `// Add extra styles here`. Copy anything new from that file to the version in your prototype, making sure you don't overwrite any extra styles you have added yourself.
+10. Open the `app/assets/sass/application.scss` file in the unzipped folder.
 
-Check the [release notes for the latest release](https://github.com/alphagov/govuk-prototype-kit/releases/latest), for any additional changes you need to make to your prototype.
+10. Copy everything down to `// Add extra styles here`, then paste it above `// Add extra styles here` in the `app/assets/sass/application.scss` file in your prototype.
 
----
+11. Check the [latest Prototype Kit release note](https://github.com/alphagov/govuk-prototype-kit/releases/latest) and follow any guidance on updating your protoype.
```

Comment:
This is tricky because they could have updated several major versions up but this will only show them the latest release note (which might not even be the last breaking one) so they could be missing out on lots of changes they should make.

Reply:
It is potentially an issue, and Joe and I discussed this ourselves too (as in the original version I'd tried to explain about following each version's release notes). On balance, we feel that if they're several versions behind, it's likely they'll hit further problems anyway. So we're best not tying ourselves in knots trying to explain about going back through versions.

Reply:
Yes it's not an easy problem to solve 😞 But it feels like we should at least try to make them aware of the release notes for earlier breaking releases.
> If your prototype is quite old, you might need to make further changes to make it work with the new kit. See [release notes](https://github.com/alphagov/govuk-prototype-kit/releases) and follow any guidance given for the versions released before your version*.  

*your version = they might need to make a note of [`VERSION.txt`](https://github.com/alphagov/govuk-prototype-kit/blob/master/VERSION.txt) before they start updating.

If the user understands that earlier versions might require them to make changes (even if they haven't made them yet), they might make that connection when they hit further problems and understand they need to go back and follow the instructions in the release notes. Trying to fix problems caused by jumping releases without understanding that you should consult release notes could be a big blocker.